src/tests/main.spec.js

```diff
@@ -0,0 +1,127 @@
+import chai from 'chai';
+import sinon from 'sinon';
+import faker from 'faker';
+import { days, defaultProduct, money, users } from './constants';
+import {showMessage, getDay, getAdultUsers, getRandomUsers, Product} from './main';
+
+const {assert, expect} = chai;
+chai.should();
+
+describe('showMessage()', () => {
+  const fakeText = 'faker.address.city';
+
+  afterEach(() => {
+    window.alert.restore();
+  });
+
+  it('alert should be invoked', () => {
+    const spy = sinon.spy(window, 'alert');
+    showMessage()
+
+    expect(spy.called).true;
+  });
+
+  it('alert should be invoked with argument', () => {
+    const spy = sinon.spy(window, 'alert');
+    showMessage(fakeText);
+
+    expect(spy.calledWith(fakeText)).true;
+  });
+});
+
+describe('getDay()', () => {
+  it('result of getDay() should be part of "days" array', () => {
+    days.indexOf(getDay()).should.be.not.equal(-1);
```

Comment:
the main purpose of `getDay()` function is to show current day. That should be tested, but not `not.equal(-1)`